### PR TITLE
Add brand name of cars into cars.txt file

### LINKED ISSUE #12: Add date to file cars.txt
Add model of cars to file cars.txt

### PR DESCRIPTION
**Pull request by MyTwoCount**
Please check this work

close #12

### CODE REVIEW

File: cars.txt [hunk 1]
```diff
@@ -1 +1 @@
-
+fiat,opel,zastava
```
Comment (LittlesProgramer) [inline cars.txt]: Please change brand opel on the seat

Comment (MyTwoCount) [reply cars.txt]: Ok

File: cars.txt [hunk 2]
```diff
@@ -1 +1 @@
-fiat,opel,zastava
+fiat,seat,zastava
```
Comment (MyTwoCount) [inline cars.txt:1]: Please check this work.

**Review by LittlesProgramer — COMMENTED** [review]
Good job.

**Review by MyTwoCount — COMMENTED** [review]
Everything is ok

**Review by MyTwoCount — COMMENTED** [review]
ok

**Comment by LittlesProgramer** [comment]
Stop this work for this issue is closed

**Pull request merged**